scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+    for each timestamp.
+- group: solution
+  name: dgnss_filter
+  type: enum
+  units:
+  default value: Fixed
+  enumerated possible values: Fixed,Float
+  Description: Determines the type of IAR the pikis will attempt to achieve.
+  Notes: If "fixed", the Piksi will output a fixed integer ambiguity estimate.  If
+    no fixed solution is available, it will rever to the float solution.  If "float",
+    the device will only output the float integer ambiguity estimate.
+- group: solution
+  name: known_baseline_n
+  type: double
+  units: meters (north)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_e
+  type: double
+  units: meters (east)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_d
+  type: double
+  units: meters (down)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: system_info
+  name: serial_number
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: The serial number of the Piksi
+  Notes: This number should match the number on the barcode and cannot be modified
+- group: system_info
+  name: firmware_version
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Indicates the firmware version for the Local Piksi
+  Notes: 'For user generated firmware, this information will appear the same as the
+    GIT command: "git describe --dirty"'
+- group: system_info
+  name: firmware_built
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Date of hardware build
+  Notes:
+- group: system_info
+  name: hw_revision
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: hardware revision for Piksi
+  Notes:
+- group: system_info
+  name: nap_version
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Version of the Navigation Acceleration Process (NAP)
+  Notes: Version of code for the NAP (which runs in the Piksi FPGA)
+- group: system_info
+  name: nap_channels
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Number of tracking channels in the NAP firmware
+  Notes:
+- group: system_info
+  name: nap_fft_index_bits
+  type:
+  units:
+  default value:
+  enumerated possible values:
+  Description: Number of bits to represent the result of fast fourier transform in
+    NAP firmware
+  Notes:
+- group: system_monitor
+  name: heartbeat_period_milliseconds
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Period for sending the SBP_HEARTBEAT and MSG_BASE_POS SBP messages
```

Comment:
I will remove the statement that this drives the MSG_BASE_POS then so we don't forget to update documentation.] when we change the MSG_BAS_POS